.github/workflows/gradle.yml

```diff
@@ -49,96 +25,108 @@ jobs:
-        git clone https://github.com/OpenLiberty/ci.common/ C:/ci.common
-        git clone https://github.com/OpenLiberty/ci.ant C:/ci.ant
-    - name: Install ci.ant and ci.common
-      run: |
-        cd C:/ci.ant
-        mvn clean install
-        cd C:/ci.common
-        mvn clean install
-    - name: Run tests with Gradle on Windows
-      working-directory: C:/ci.gradle
-      # LibertyTest is excluded because test0_run hangs
-      run: ./gradlew clean install check -P"test.exclude"="**/*15*,**/Polling*,**/TestLoose*,**/LibertyTest*" -Druntime=${{ matrix.RUNTIME }} -DruntimeVersion="${{ matrix.RUNTIME_VERSION }}" ${GRADLE_OPTS} --stacktrace --info --no-daemon
-      timeout-minutes: 60
-      env:
-        GRADLE_OPTS: "-Dorg.gradle.jvmargs='-XX:MaxMetaspaceSize=512m'"
-    - name: Copy build/report/tests/test for upload
-      if: ${{ always() }}
-      working-directory: C:/ci.gradle
-      run: cp -r build/reports/tests/test D:/buildReports/${{runner.os}}/java${{matrix.java}}/${{matrix.RUNTIME}}-${{matrix.RUNTIME_VERSION}}/      
-    - uses: actions/upload-artifact@v2
-      if: ${{ failure() }}
-      with:
-        name: buildReportsArtifact
-        path: D:/buildReports
-        retention-days: 3
-  build-unix:
-    runs-on: ubuntu-18.04
-    name: ${{ matrix.RUNTIME }} ${{ matrix.RUNTIME_VERSION }}, Java ${{ matrix.java }}, Linux
-    needs: cache
-    # Steps represent a sequence of tasks that will be executed as part of the job
-    strategy:
-      fail-fast: false
-      matrix:
-        # test against latest update of each major Java version, as well as specific updates of LTS versions:
-        RUNTIME: [ol, wlp]
-        RUNTIME_VERSION: [21.0.0.3, 20.0.0.12]
-        java: [11, 8]
-        exclude:
-        - java: 8
-          RUNTIME_VERSION: 20.0.0.12
-    steps:
     - name: Setup Java ${{ matrix.java }}
-      uses: joschi/setup-jdk@v2
+      uses: actions/setup-java@v2
       with:
+        distribution: 'adopt'
         java-version: ${{ matrix.java }}
-    - name: Checkout ci.gradle
-      uses: actions/checkout@v2
-    - name: Checkout ci.common
+        cache: 'gradle'
+    - name: Checkout ci.common (Linux)
+      if: ${{ runner.os == 'Linux' }}
       uses: actions/checkout@v2
       with:
         repository: OpenLiberty/ci.common
         path: ci.common
-    - name: Checkout ci.ant
+    - name: Checkout ci.ant (Linux)
+      if: ${{ runner.os == 'Linux' }}
       uses: actions/checkout@v2
       with:
         repository: OpenLiberty/ci.ant
         path: ci.ant
-    - name: Install ci.ant and ci.common
+    # Moving and cloning to C: drive for Windows for more disk space
+    - name: Clone ci.ant, ci.common, ci.gradle repos to C drive (Windows)
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        cp -r D:/a/ci.gradle/ci.gradle C:/ci.gradle
+        git clone https://github.com/OpenLiberty/ci.common/ C:/ci.common
+        git clone https://github.com/OpenLiberty/ci.ant C:/ci.ant
```

Comment:
Why does one https path end with `/` and the other one doesn't? Also, when I do a git clone with https it usually ends with `.git`. Why is this different?

Reply:
I didn't even notice either of these while rearranging items.

I think the trailing slash doesn't matter at the end of these URLs, but I can clean it up.
and git will automatically resolve the repository, and will look for the `.git` version if it can't find it.

Reply:
I added these. It was simple oversight on both comments. I manually typed the git URLs from memory and forgot to correct them. 
Although it performs the same, it would be more correct to clone with the `.git` extension